neuro_san/session/http_concierge_session.py

```diff
@@ -0,0 +1,45 @@
+
+# Copyright (C) 2023-2025 Cognizant Digital Business, Evolutionary AI.
+# All Rights Reserved.
+# Issued under the Academic Public License.
+#
+# You can be released from the terms, and requirements of the Academic Public
+# License by purchasing a commercial license.
+# Purchase of a commercial license is mandatory for any use of the
+# neuro-san SDK Software in commercial settings.
+#
+# END COPYRIGHT
+
+from typing import Any
+from typing import Dict
+
+import json
+import requests
+
+from neuro_san.interfaces.concierge_session import ConciergeSession
+from neuro_san.session.abstract_http_service_agent_session import AbstractHttpServiceAgentSession
+
+
+class HttpConciergeSession(AbstractHttpServiceAgentSession, ConciergeSession):
+    """
+    Implementation of ConciergeSession that talks to an HTTP service.
+    This is largely only used by command-line tests.
+    """
```

Comment:
Round out the connection offerings for the ConciergeSession.
We already had something for gRPC.